crm_phonecall/models/crm_phonecall.py

```diff
@@ -139,6 +139,7 @@ def schedule_another_phonecall(self, schedule_time, call_summary,
                                    tag_ids=False, action='schedule'):
         """
         action :('schedule','Schedule a call'), ('log','Log a call')
+        TODO: return a recordset instead of a dict
```

Comment:
I think the fix is much easier if you just do this. Simply add a param `return_recordset=False`, and return a recordset if it is `True`, to keep backwards compatibility. Then add to Roadmap a note about removing that feature and always returning a recordset, as a good migration reminder.